docs/administration/security-hub/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Security Hub
+weight: 43
+---
+
+The Security Hub feature in Harbor offers administrators a comprehensive and centralized view of the current security status of artifacts stored within the Harbor registry.
+
+You can access the Security Hub feature  
+via the Harbor UI Navigation or direct URL:
+
+1. Method 1: Accessing the Security Hub via UI Navigation
+
+   - Login to Harbor with admin user. locate and click on the "Administration" option in the Harbor UI.
+
+   - Within the Administration section, find and click on the "Interrogation Service" section.
+
+   - On the "Interrogation Service" settings page, locate the "Security Hub" tab and click on it. This will open the Security Hub dashboard.
+
+2. Method 2: Accessing the Security Hub via Direct URL:
+
+   - Open your web browser, and enter the following URL in the address bar: `https://<harbor-domain>/harbor/interrogation-services/security-hub`.
+ 
+   Note: Replace <harbor-domain> with the appropriate domain or IP address of your Harbor instance
+
+The page title displays total count of artifacts and scanned artifacts.
+
+## Total Vulnerabilities
+
+The first card shows the total number of vulnerabilities found in the scanned images. the count is grouped by the severity of the vulnerabilities.
```

Comment:
The first card shows the total number of vulnerabilities found in the scanned images. The total number is broken up into severity groups with their respective occurrences.

Reply:
Updated